src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -1,36 +1,299 @@
+        FileUtils.touch(new File(fullFilePath));
+    }
+
+    public void setPaymentImportDirectory(String paymentImportDirectory) {
+        this.paymentImportDirectory = paymentImportDirectory;
+    }
+
+    public String getPaymentImportDirectory() {
+        return this.paymentImportDirectory;
+    }
+
+    public ParameterService getParameterService() {
+        return parameterService;
+    }
+
+    public void setParameterService(ParameterService parameterService) {
+        this.parameterService = parameterService;
+    }
+
+    public DataDictionaryService getDataDictionaryService() {
+        return dataDictionaryService;
+    }
+
+    public void setDataDictionaryService(DataDictionaryService dataDictionaryService) {
+        this.dataDictionaryService = dataDictionaryService;
+    }
+
+    public ConcurRequestedCashAdvanceService getConcurRequestedCashAdvanceService() {
+        return concurRequestedCashAdvanceService;
+    }
+
+    public void setConcurRequestedCashAdvanceService(ConcurRequestedCashAdvanceService concurRequestedCashAdvanceService) {
+        this.concurRequestedCashAdvanceService = concurRequestedCashAdvanceService;
+    }
+
+    public CUMarshalService getCuMarshalService() {
+        return cuMarshalService;
+    }
+
+    public void setCuMarshalService(CUMarshalService cuMarshalService) {
+        this.cuMarshalService = cuMarshalService;
+    }
+
+    public DateTimeService getDateTimeService() {
+        return DateTimeService;
+    }
+
+    public void setDateTimeService(DateTimeService dateTimeService) {
+        DateTimeService = dateTimeService;
+    }
+
+    public int getPayeeNameFieldSize() {
+        if (payeeNameFieldSize == 0) {
+            payeeNameFieldSize = retrievePayeeNameFieldSize();
+        }
+        return payeeNameFieldSize;
+    }
+
+    private PdpFeedFileBaseEntry buildPdpFeedBaseEntry(ConcurRequestExtractFile requestExtractFile) {
+        int totalPdpDetailRecordsCount = 0;
+        KualiDecimal totalPdpDetailRecordsAmount = KualiDecimal.ZERO;
+        PdpFeedFileBaseEntry pdpBaseEntry = new PdpFeedFileBaseEntry();
+        pdpBaseEntry.setHeader(buildPdpFeedHeaderEntry(requestExtractFile.getBatchDate()));
+        List<PdpFeedGroupEntry> groupEntries = new ArrayList<PdpFeedGroupEntry>();
+        for (ConcurRequestExtractRequestDetailFileLine detailFileLine : requestExtractFile.getRequestDetails()) {
+            if (detailFileLine.getValidationResult().isCashAdvanceLine() && detailFileLine.getValidationResult().isValidCashAdvanceLine()) {
+                PdpFeedDetailEntry pdpDetailEntry = buildPdpFeedDetailEntry(detailFileLine, buildPdpFeedAccountingEntry(detailFileLine));
+                List<PdpFeedDetailEntry> pdpDetailEntries = new ArrayList<PdpFeedDetailEntry>();
+                pdpDetailEntries.add(pdpDetailEntry);
```

Comment:
No I cannot.  I need to reference attributes of the pdpDetailEntry on line 147 below.